applications/utilities/reconstructPointDistribution/reconstructPointDistribution.C

```diff
@@ -129,14 +129,19 @@ int main(int argc, char *argv[])
         fileName probeDir;
         fileName probeSubDir = phaseName + "Probes";
         probes mProbes(probeSubDir, runTime, probesDict);
+        #if (OPENFOAM > 2506)
+        const label nProbes = mProbes.probeModel().size();
+        #else
+        const label nProbes = mProbes.size();
+        #endif
 
         if (mesh.name() != polyMesh::defaultRegion)
         {
             probeSubDir = probeSubDir/mesh.name();
         }
 
         probeSubDir = "postProcessing"/probeSubDir/mesh.time().timeName();
```

Comment:
Not for Kuti, but for Alberto:

Can simplify this:
```
if (mesh.name() != polyMesh::defaultRegion)
{
    probeSubDir = probeSubDir/mesh.name();
}
probeSubDir = "postProcessing"/probeSubDir/mesh.time().timeName();
```
as 
```
probeSubDir = "postProcessing"/probeSubDir/mesh.regionName()/mesh.time().timeName();
```
which will filter out the `region0` stuff automatically.